### PR TITLE
Update Chinese translation

### LINKED ISSUE #61: Localization update
PRs #57 and #63 introduce [new UI strings](https://github.com/fpvout/DigiView-Android/pull/57/commits/25ab31c87bcaa6496cc08810be5f360b03037e3b#diff-5e01f7d37a66e4ca03deefc205d8e7008661cdd0284a05aaba1858e6b7bf9103) that need to be translated to existing languages (German, Spanish, French and Chinese).

### PR DESCRIPTION
**Pull request by DusKing1**

### CODE REVIEW

**Comment by DusKing1** [comment]
Wait a minute, why I can't see other strings in my xml?

File: app/src/main/res/values-zh/strings.xml [hunk 1]
```diff
@@ -1,14 +1,45 @@
...
+    <string name="video_player">视频播放器</string>
+    <string name="full_screen">全屏</string>
+    <string name="full_screen_summary">也可以通过在视频播放器上点按两次或双指捏合来进行切换。</string>
+    <string name="show_watermark">显示 DigiView 水印</string>
+    <string name="links">相关链接</string>
+    <string name="our_website">我们的网站</string>
+    <string name="discord_summary">与我们和其他 DigiView 用户聊天</string>
+    <string name="digiview_web_app">DigiView Web应用</string>
+    <string name="digiview_web_app_summary">可以在Chrome和基于Chrome的浏览器中使用，甚至可以在Android上使用！</string>
+    <string name="github_summary"我们所有的代码都是开源的</string>
```
Comment (DusKing1) [inline app/src/main/res/values-zh/strings.xml]: miss a >

File: app/src/main/res/values-zh/strings.xml [hunk 2]
```diff
@@ -1,14 +1,45 @@
...
+    <string name="privacy_policy_summary">查看我们收集数据的内容以及我们如何使用它们</string>
+    <string name="video_player">视频播放器</string>
+    <string name="full_screen">全屏</string>
+    <string name="full_screen_summary">也可以通过在视频播放器上点按两次或双指捏合来进行切换。</string>
+    <string name="show_watermark">显示 DigiView 水印</string>
+    <string name="links">相关链接</string>
+    <string name="our_website">我们的网站</string>
+    <string name="discord_summary">与我们和其他 DigiView 用户聊天</string>
+    <string name="digiview_web_app">DigiView Web应用</string>
+    <string name="digiview_web_app_summary">可以在Chrome和基于Chrome的浏览器中使用，甚至可以在Android上使用！</string>
```
Comment (PenguinK) [inline app/src/main/res/values-zh/strings.xml:37]: maybe "可以在Chrome以及基于Chrome核心的浏览器中使用，甚至可以在Android上运行！"？

**Comment by jlucidar** [comment]
Cool ! thanks for the PR :) @PenguinK I'm merging this, feel free to open a new PR with the modifications you suggested :)

**Comment by DusKing1** [comment]
LOL @PenguinK  I just wanna say I'll fix this and you @jlucidar  just give it merged!

File: app/src/main/res/values-zh/strings.xml (hunk 2 above)
Comment (DusKing1) [reply app/src/main/res/values-zh/strings.xml:37]: Nice catch!

**Comment by PenguinK** [comment]
nice job! @DusKing1

**Pull request merged**